code/game/turfs/walls/wall_types.dm

```diff
@@ -465,3 +465,71 @@
 		'icons/turf/walls/siding_red_2.dmi',
 		'icons/turf/walls/siding_red_3.dmi',
 	)
+
+/turf/closed/wall/urban
+	name = "bare metal walls"
+	desc = "A thick and chunky metal wall. The surface is barren and imposing."
+	icon = 'icons/turf/walls/urban_wall_regular.dmi'
+	icon_state = "urban_wall_regular-0"
+	walltype = "wall"
+	base_icon_state = "urban_wall_regular"
+
+/turf/closed/wall/urban/colony/ribbed
+	name = "bare metal walls"
+	desc = "A thick and chunky metal wall. The surface is barren and imposing."
+	icon = 'icons/turf/walls/hybrisa_colony_walls.dmi'
+	icon_state = "wall-reinforced"
+	walltype = "wall"
+	base_icon_state = "hybrisa_colony_walls"
+
+/turf/closed/wall/urban/colony/hospital
+
+/turf/closed/wall/urban/colony/reinforced
+
+/turf/closed/wall/urban/colony/office
+
+/turf/closed/wall/urban/spaceport_unmeltable
+
+/turf/closed/wall/urban/spaceport_reinforced
+
+/turf/closed/wall/urban/engineering
+
+/turf/closed/wall/urban/colony/engineering/ribbed
+	name = "bare metal walls"
+	desc = "A thick and chunky metal wall. The surface is barren and imposing."
+	icon = 'icons/turf/walls/hybrisa_colony_walls.dmi'
+	icon_state = "wall-reinforced"
+	walltype = "wall"
+	base_icon_state = "hybrisa_colony_walls"
+
+/turf/closed/wall/urban/research/reinforced
+
+/turf/closed/wall/urban/colony/reinforced/hull
+
+/turf/closed/wall/urban/marhsalls_reinforced
+
+/turf/closed/wall/urban/colony/engineering/reinforced
+
+/turf/closed/wall/urban/spaceport
+
+/turf/closed/wall/urban/research/reinforced/hull
+
+/turf/closed/wall/urban/marhsalls_unmeltable
+
+/turf/closed/wall/urban/colony/engineering/reinforced/hull
+
```

Comment:
whats with all these subtypes with zero differences?

Reply:
This is not me, this is the original map author declaring a million and a half walls that don't do anything. I've been consolidating them but haven't finished yet.